pycqed/analysis_v2/timedomain_analysis.py

```diff
@@ -429,103 +523,24 @@ def create_meas_results_per_qb(self):
     def process_data(self):
         super().process_data()
 
-        self.data_with_reset = False
-        if self.data_filter is None:
-            if 'active' in self.prep_params.get('preparation_type', 'wait'):
-                reset_reps = self.prep_params.get('reset_reps', 1)
-                self.data_filter = lambda x: x[reset_reps::reset_reps+1]
-                self.data_with_reset = True
-            elif "preselection" in self.prep_params.get('preparation_type',
-                                                        'wait'):
-                self.data_filter = lambda x: x[1::2]  # filter preselection RO
-        if self.data_filter is None:
-            self.data_filter = lambda x: x
-
         self.create_sweep_points_dict()
         self.create_meas_results_per_qb()
-
-        # temporary fix for appending calibration points to x values but
```

Comment:
I removed it because it seemed like a merge bug. The comment wasn't relevant to the lines of code above or below. It seems like it used to belong to the cal points extraction lines. But now I don't find any temporary fix of this sort and in this cleanup effort I make sure we always handle correctly this extension based on cal points (or at least I hope so ... )